local/pipeline.go

```diff
@@ -42,15 +42,23 @@ func (s *step) UnmarshalJSON(data []byte) error {
 		branches = b
 	}
 
-	// Handle various types of branch vs branches
 	if branches != nil {
+		// Sometimes branches are space delimited, sometimes comma
```

Comment:
BK backend doesn't support `branches: "foo, bar"` - is that what this bit is for? Or is this something else?

Reply:
Ok cool, not sure where I got that from, but will ditch it.